source/puppet/4.3/reference/puppet_collections.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: default
+title: "About Puppet Collections and Packages"
+canonical: "/puppet/latest/reference/puppet_collections.html"
+---
+
+{% include puppet-collections/_puppet_collections_intro.md %}
+
+## Puppet Collection contents
+
+Puppet Collection 1 contains the following components:
+
+{% include puppet-collections/_puppet_collection_1_contents.md %}
+
+## Using Puppet Collections
+
+{% include puppet-collections/_puppet_collections_using.md %}
+
+### Yum-based systems
+
+{% include puppet-collections/_puppet_collection_1_yum.md %}
+
+{% include puppet-collections/_puppet_collection_1_el7.md %}
+
+{% include puppet-collections/_puppet_collection_1_el6.md %}
+
+{% include puppet-collections/_puppet_collection_1_el5.md %}
+
+{% include puppet-collections/_puppet_collection_1_f22.md %}
+
+{% include puppet-collections/_puppet_collection_1_f21.md %}
+
+### Apt-based systems
+
+{% include puppet-collections/_puppet_collection_1_apt.md %}
+
+{% include puppet-collections/_puppet_collection_1_u1504.md %}
+
+{% include puppet-collections/_puppet_collection_1_u1404.md %}
+
+{% include puppet-collections/_puppet_collection_1_u1204.md %}
+
+{% include puppet-collections/_puppet_collection_1_d8.md %}
+
+{% include puppet-collections/_puppet_collection_1_d7.md %}
+
+{% include puppet-collections/_puppet_collection_1_d6.md %}
+
+### OS X systems
+
+{% include puppet-collections/_puppet_collection_1_osx.md %}
+
+{% include puppet-collections/_puppet_collection_1_osx1011.md %}
+
+{% include puppet-collections/_puppet_collection_1_osx1010.md %}
+
+{% include puppet-collections/_puppet_collection_1_osx1009.md %}
+
+## Verifying Puppet packages
```

Comment:
The content from this point forward supplants the unversioned Package Verification guide. This content is stable, but can be tailored as needed in past or future versions that might add or change this process.